src/views/checkups/network/components/form/CheckupsNetworkFormNads.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Dispatch, FC, SetStateAction, SyntheticEvent, useState } from 'react';
+
+import useNADsData from '@kubevirt-utils/components/NetworkInterfaceModal/components/hooks/useNADsData';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
+import { Dropdown, DropdownItem, DropdownToggle, FormGroup } from '@patternfly/react-core';
+
+type CheckupsNetworkFormNadsProps = {
+  selectedNad: string;
+  setSelectedNad: Dispatch<SetStateAction<string>>;
+};
+const CheckupsNetworkFormNads: FC<CheckupsNetworkFormNadsProps> = ({
+  selectedNad,
+  setSelectedNad,
+}) => {
+  const { t } = useKubevirtTranslation();
+  const [namespace] = useActiveNamespace();
+  const { nads } = useNADsData(namespace);
+  const [isOpenDropdown, setIsOpenDropdown] = useState<boolean>(false);
+
+  const nadsDropdownItems = (nads || [])?.map((nad) => (
+    <DropdownItem key={nad?.metadata?.uid}>{nad.metadata.name}</DropdownItem>
+  ));
+
+  return (
+    <FormGroup fieldId="nad" isRequired label={t('NetworkAttachmentDefinition')}>
+      <Dropdown
+        onSelect={(e: SyntheticEvent<HTMLDivElement, Event>) => {
+          setSelectedNad((e?.target as HTMLElement)?.innerText);
+          setIsOpenDropdown(false);
+        }}
+        toggle={
+          <DropdownToggle
+            onToggle={() => {
+              setIsOpenDropdown((value) => !value);
+            }}
+            id="toggle-nad"
+          >
+            {selectedNad || t('Select NetworkAttachmentDefinition')}
+          </DropdownToggle>
+        }
+        className="CheckupsNetworkForm--main__nads"
+        disabled={isEmpty(nads)}
+        dropdownItems={nadsDropdownItems}
+        id="nad"
+        isOpen={isOpenDropdown}
+      />
```

Comment:
I would use a Select with Typeahead, so that the user can search the right NAD in situations where there are too much